docs/test/layouts/product-preview.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Product Preview Test Page
+layout: ProductDiscover
+frameOverrides:
+  header:
+    homeLink: /mosaic/index
+    title: Mosaic (BETA)
+    logo: /img/favicon.png
+    searchNamespace: mosaic
+    menu:
+      - title: Docs
+        link: /mosaic/docs/index
+      - title: Quick Start
+        link: /mosaic/quick-start/index
+  footer:
+    description: Footer Content
+    title: Product Preview Layout Footer
+    href: /mosaic
+---
+
+## {meta.title}
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
```

Comment:
FYI
Lorem ipsum  blows up the spellchecker, if we intend to  use that

Reply:
doh...i will chnage it over then

Reply:
can we exclude these files from the spellchecker?